src/core/tsi/ssl_transport_security.c

```diff
@@ -44,6 +44,9 @@
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
+#if defined(OPENSSL_IS_BORINGSSL)
```

Comment:
Is there any harm in including that all the time? I'm not a fan of these types of `#if` if they can be avoided.